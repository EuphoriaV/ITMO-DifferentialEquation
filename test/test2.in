1
((((x*x)-1)*(2*x))+4)
3
-1