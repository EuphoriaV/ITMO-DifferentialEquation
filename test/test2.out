7.5178473